test/unit/api/regular/wide.cpp

```diff
@@ -72,7 +72,7 @@ TTS_CASE_TPL( "Check eve::wide splat constructor", eve::test::simd::all_types)
 
   // Test smaller size wide for non-garbage
   using v_t = typename T::value_type;
-  if constexpr( T::size() < eve::fundamental_cardinal_v<v_t> && !eve::has_emulated_abi_v<T> )
```

Comment:
this comes from the fundamental wide being one idea I think.

Reply:
For RISC-V I forbid this as we can have different types as storage type depending on cardinal value. To support this I need to add additional constructor in core wide functionality.